packages/components/src/components/CodeEditor.tsx

```diff
@@ -90,6 +91,7 @@ export const CodeEditor = forwardRef<CodeEditorHandle, CodeEditorProps>(
       errors = [],
       sourceId,
       project,
+      lineWrapping = false,
```

Comment:
I'd be happy with `true` default; I have it enabled in my VS Code (but I think it's not the default there?).

Or even with no option, just line wrap always.

But I'm not sure if everyone would like that.